base/multidimensional.jl

```diff
@@ -1466,7 +1466,7 @@ end
 
 ## Permute array dims ##
 
-function permutedims(B::StridedArray, perm)
+function permutedims(B::StridedArray, perm = (2,1))
```

Comment:
Perhaps better to provide a default `(2, 1)` perm only for two-dimensional `StridedArray`s? For example, perhaps add the short child
```julia
permutedims(B::StridedMatrix) = permutedims(B, (2, 1))
```
rather than modify this general method's signature?

Reply:
I thought of that at first, but I note that you get a pretty coherent error message for 3D arrays as this is.